test/spec/features/copy-paste/BpmnCopyPasteSpec.js

```diff
@@ -71,6 +71,17 @@ describe('features/copy-paste', function() {
       }));
 
 
+      it('should copy boundary events without host', inject(function() {
```

Comment:
I'd prefer a full integration test here. Copy, paste & verify the result.

Reply:
Ah yes, good idea, I'm on it!